SU-HSDO/stanford_mrc
# MRCD8-51 Focal point on media

# READY FOR REVIEW
# Summary
- Enabled focal point for media.

# Needed By (Date)
- End of sprint

# Urgency
- Medium

# Steps to Test

1. update composer (new dependency)
2. sync with prod
3. checkout this branch
4. update database
5. create any content with a wysiwyg
6. open the media browser and upload a new image (preferably one that is not square. something really tall or really wide)
7. ensure the focal point selector is on the image thumbnail
8. continue to embed it and choose the "Events" image style
9. ensure the image is cropped to the configured point as chosen earlier.
10. save that piece of content with the image embeded
11. navigate to the media item and change the focal point. 
12. reload the node you created and ensure the image crop changed.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
